.github/workflows/compressed-size-action.yml

```
@@ -10,7 +10,7 @@ jobs:
       - uses: pnpm/action-setup@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 22
```

What about using `lts/*` instead? so we always get the latest lts version

```suggestion
          node-version: lts/*
```

---

@dbritto-dev Sounds good 👍